docker-compose-host.yml.template

```diff
@@ -1,14 +1,17 @@
-version: '2'
+version: '3.3'
 services:
   plex:
     container_name: plex
     image: plexinc/pms-docker
+    ports:
+      - "32400"
     restart: unless-stopped
     environment:
-      - TZ=<timezone>
-      - PLEX_CLAIM=<claimToken>
+      - TZ=${TIMEZONE}
+      - PLEX_CLAIM=${CLAIM_TOKEN}
+      - PLEX_HOME=${PLEX_HOME}
     network_mode: host
     volumes:
-      - <path/to/plex/database>:/config
-      - <path/to/transcode/temp>:/transcode
-      - <path/to/media>:/data
+      - ${PLEX_HOME}/config:/config
+      - ${PLEX_HOME}/transcode:/transcode
+      - ${PLEX_HOME}/data:/data
```

Comment:
Far too constraining to force all of these directories into the same directory tree and to force these specific directory names.

Reply:
You are correct, this should be changed.  I will break it out.